.github/workflows/Build_and_publish_package.yml

```diff
@@ -0,0 +1,35 @@
+name: Build and Deploy Packages
+
+on: [push]
+
+env:
+  # Customize the CMake build type here (Release, Debug, RelWithDebInfo, etc.)
+  BUILD_TYPE: RelWithDebInfo
+jobs:
+  build:
+    # The CMake configure and build commands are platform agnostic and should work equally
+    # well on Windows or Mac.  You can convert this to a matrix build if you need
+    # cross-platform coverage.
+    # See: https://docs.github.com/en/actions/configuring-and-managing-workflows/configuring-a-workflow#configuring-a-build-matrix
+    runs-on: windows-latest
+
+    steps:
+    # Checkout repo into build area
+    - uses: actions/checkout@v2
+      with:
+        submodules: recursive   
+    - name: Build project
+      # Build project using powershell script
+      run: |
+        .\build.libgit2.ps1
+        .\buildpackage_forMendix.ps1 -version 1.101.1
+      shell: powershell
+      # Publish the package to github actions
+      # We add the github source so that we can authenticate correctly to github packages with both a username and a password
+    - name: Publish package      
+      run: |
+        nuget source Add -Name "GitHub" -Source "https://nuget.pkg.github.com/MarcellvanRooyen/index.json" -UserName MarcellVanRooyen -Password ${{ secrets.GHPACKAGESTOKEN }}
```

Comment:
This source still refers to your personal project

Reply:
updated

Reply:
updated to use https://nuget.pkg.github.com/Mendix/index.json